include/morphio/properties.h

```diff
@@ -28,10 +28,11 @@ namespace Property {
 template <typename T>
 void _appendVector(std::vector<T>& to, const std::vector<T>& from, int offset);
 extern template void _appendVector(std::vector<Point>& to, const std::vector<Point>& from, int
-offset); extern template void _appendVector(std::vector<float>& to, const std::vector<float>& from,
-int offset); extern template void _appendVector(std::vector<unsigned int>& to, const
-std::vector<unsigned int>& from, int offset); template <typename T> std::vector<typename T::Type>
-copySpan(const std::vector<typename T::Type>& data, SectionRange range);
+offset); extern template void _appendVector(std::vector<morphio::floatType>& to, const
+std::vector<morphio::floatType>& from, int offset); extern template void
+_appendVector(std::vector<unsigned int>& to, const std::vector<unsigned int>& from, int offset);
+template <typename T> std::vector<typename T::Type> copySpan(const std::vector<typename T::Type>&
+data, SectionRange range);
```

Comment:
unnecessary reformat?

Reply:
clang-format script from the repo (enforced by CI)

Reply:
removed commented out code